packages/@react-spectrum/tag/docs/TagGroup.mdx

```diff
@@ -154,3 +164,113 @@ function Example() {
   </Item>
 </TagGroup>
 ```
+
+### With avatars
+
+```tsx example
+<TagGroup aria-label="TagGroup with avatars example">
+  <Item>
+    <Avatar src="https://i.imgur.com/kJOwAdv.png" alt="Person 1 avatar" />
```

Comment:
I think these avatars should not have alt text (it defaults to "" in the Avatar component). Right now tags are read by screen readers as "Person 1 avatar Person 1", which is kinda unnecessarily verbose. The avatar is essentially decorative so I don't think it needs its own alt text.